### PR TITLE
Prerelease v1.10.0

### PR DESCRIPTION
**Pull request by MadVikingGod**
A step in #3157 

Releases https://github.com/open-telemetry/opentelemetry-go/milestone/26

### Added

- Support Go 1.19. (#3077)
  Include compatibility testing and document support. (#3077)
- Support the OTLP ExportTracePartialSuccess response; these are passed to the registered error handler. (#3106)
- Upgrade go.opentelemetry.io/proto/otlp from v0.18.0 to v0.19.0 (#3107)

### Changed

- Fix misidentification of OpenTelemetry `SpanKind` in OpenTracing bridge (`go.opentelemetry.io/otel/bridge/opentracing`).  (#3096)
- The exponential histogram mapping functions have been updated with
  exact upper-inclusive boundary support following the [corresponding
  specification change](https://github.com/open-telemetry/opentelemetry-specification/pull/2633). (#2982)
- Attempting to start a span with a nil `context` will no longer cause a panic. (#3110)
- All exporters will be shutdown even if one reports an error (#3091)

### CODE REVIEW

**Review by Aneurysm9 — APPROVED** [review]

**Comment by MrAlias** [comment]
> The exponential histogram mapping functions have been updated with
> exact upper-inclusive boundary support following the corresponding
> specification change. (Change the inclusivity of exponential histogram bounds #2982)

This is not included in this release and needs to be removed from the changelog entry.

**Comment by jmacd** [comment]
> This is not included in this release and needs to be removed from the changelog entry.

What happened to this change? I've been waiting to refer to it from opentelemetry-collector.

**Comment by jmacd** [comment]
I'd like to suggest we include the exponential histogram mapping function change in this release,
https://github.com/open-telemetry/opentelemetry-go/pull/3159

**Comment by MrAlias** [comment]
> @MrAlias Did I miss something? Maybe this needs to be a v1.10, and metrics/v0.32. release?

We had agreed the next metric release is going to be for the alpha SDK.  I think the versioning is correct as is.



> > This is not included in this release and needs to be removed from the changelog entry.
> 
> What happened to this change? I've been waiting to refer to it from opentelemetry-collector, here: [open-telemetry/opentelemetry-collector#5938](https://github.com/open-telemetry/opentelemetry-collector/pull/5938)

I thought it was agreed the commit hash version of main would be used as a dependency for this package?

**Comment by jmacd** [comment]
> I thought it was agreed the commit hash version of main would be used as a dependency for this package?

I can do that. I've done that in `otel-launcher-go` but I don't feel comfortable doing that for the OTel collector. I proposed #3159 because it seems to avoid leaving clutter in the `sdk/metric/aggregator` package where the new SDK doesn't want it to belong, it's fine to let this go to another release.

**Review by jmacd — APPROVED** [review]
I'd prefer if #3156 could be included in this release, approved either way.

**Comment by MadVikingGod** [comment]
I don't remember the v0.32 discussion, but I'm fine with that.  

I have reverted the changes for those versions (v0.32.0), and have removed the changelog entry.

**Comment by MadVikingGod** [comment]
As #3156 is also a customer impacting bugfix I also would like to see that in this release.  I will delay until that is landed in main, and can be added to this PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,20 +8,20 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [1.10.0] - 2022-09-09
```
[the remark this answers is not in the document]
Comment (MadVikingGod) [reply CHANGELOG.md:11]: Should we keep that? We aren't going to release a version with that change.  

If we don't release another version of the old metrics SDK then that change will never make it to any tag.

**Review by MrAlias — APPROVED** [review]

**Review by jmacd — APPROVED** [review]

**Pull request merged**